sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -76,13 +70,26 @@
      */
     private KeyVaultClient keyVaultClient;
 
+    /**
+     * Store certificates on portal info
+     */
+    private final KeyVaultCertificates keyVaultCertificates;
+
+    private static final boolean REFRESH_CERTIFICATES_WHEN_HAVE_UN_TRUST_CERTIFICATE = Optional.ofNullable(System.getProperty("azure.keyvault.jca.refresh-certificates-when-have-un-trust-certificate"))
```

Comment:
Please don't use `static`.

If we use `static`, then `KeyVaultCertificatesEnvironmentPostProcessor#putEnvironmentPropertyToSystemProperty` will not take effect.

And please add the 2 property in 
1. `KeyVaultCertificatesEnvironmentPostProcessor#putEnvironmentPropertyToSystemProperty` .
2. `KeyVaultProperties`.